dashboard/src/shared/Config.ts

```diff
@@ -1,20 +1,15 @@
 import axios from "axios";
 
-interface IConfig {
+export interface IConfig {
   namespace: string;
 }
 
 export default class Config {
   public static async getConfig() {
-    if (Config.config) {
-      return Config.config;
-    }
     const url = `${Config.APIEndpoint}`;
     const { data } = await axios.get<IConfig>(url);
-    Config.config = data;
-    return Config.config;
+    return data;
   }
 
-  private static config: IConfig;
   private static APIEndpoint: string = "/config.json";
```

Comment:
It's not related to this PR but this is a bit confusing (using `axios.get` with an "URL" to read a local file). Mostly because this action is now unnecessary async.

Reply:
@andresmgot perhaps you're misunderstanding this a bit? The nginx server serves the config.json file, it's not embedded in the React application. I'm not sure what you mean by local file, at some point the React app has to fetch this from the server.

Reply:
I think that the key here is that the React app is just a bundle of static files so AFAIK there is no way of exposing this env variable without triggering a build (via webpack) at deployment time.

If there was a way for Nginx to expose this env variable as a header that could be an alternative. Although I have not found any example nor insights on how to achieve so.

Reply:
No problem, thanks for asking!